Sources/Networking/Responses/CustomerCenterConfigResponse.swift

```diff
@@ -76,6 +76,12 @@ struct CustomerCenterConfigResponse {
             let title: String
             let subtitle: String
             let productMapping: [String: String]
+            let crossProductPromotions: [String: CrossProductPromotion]?
+
+            struct CrossProductPromotion {
+                let storeOfferIdentifier: String
+                let targetProductId: String
+            }
```

Comment:
I think I would define this at the same level of PromotionalOffer at least. This level of nesting is unnecessary